src/dispatch/Dispatch.ts

```diff
@@ -101,7 +102,7 @@ export class Dispatch {
      * Send meta data and events to the AWS RUM data plane service via fetch.
      */
     public dispatchFetch = async (): Promise<{ response: HttpResponse }> => {
-        return this.dispatch(this.rum.sendFetch);
+        return this.dispatch(this.rum.sendFetch).catch(this.handleReject);
```

Comment:
question: for automatic dispatches that will occur at a predefined interval, we will fail silently - however, if the retries failed after invoking dispatch directly from `orchestration`, we are not failing silently.
```
public dispatch(): void {
        this.dispatchManager.dispatchFetch();
    }
``` 

is this intended?

Reply:
I **think** my reasoning here was that if you're manually invoking it you may want to know if it fails.

There's a ticket somewhere to revisit our error handling strategy, but for this change it is out of scope.